types/webpack-assets-manifest/webpack-assets-manifest-tests.ts

```diff
@@ -0,0 +1,106 @@
+import WebpackAssetsManifest from "webpack-assets-manifest";
```

Comment:
The module doesn't have a default export so this import is incorrect. Use `import { export } from 'module'` instead

Reply:
@armanio123 The module is exported as a CommonJS module with `export = WebpackAssetsManifest;`. Since [`esModuleInterop`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#support-for-import-d-from-cjs-form-commonjs-modules-with---esmoduleinterop) is set to `true` in tsconfig.json, the import works.

Reply:
My bad, I didn't noticed that you have the option enabled.

Reply:
Never mind. Thanks for the review and commit!